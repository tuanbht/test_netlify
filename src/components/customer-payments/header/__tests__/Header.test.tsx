import { shallow } from 'enzyme';
import React from 'react';
import Header from '../index';

describe('Header', () => {
  it('renders template correctly', () => {
    expect(shallow(<Header canBack={true} goBack={jest.fn()} />)).toMatchSnapshot();
  });
});
