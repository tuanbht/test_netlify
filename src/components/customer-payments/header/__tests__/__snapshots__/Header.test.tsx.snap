// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header can back is false renders template without arrow back button 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-title-2"
  >
    <ForwardRef(SvgPalomaLogo)
      className="makeStyles-logo-4"
    />
    <span>
      Paloma Pay
    </span>
  </div>
</WithStyles(ForwardRef(Container))>
`;

exports[`Header can back is true renders template with arrow back button 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-root-1"
>
  <ArrowBackIcon
    className="makeStyles-goBack-3"
    onClick={[MockFunction]}
  />
  <div
    className="makeStyles-title-2"
  >
    <ForwardRef(SvgPalomaLogo)
      className="makeStyles-logo-4"
    />
    <span>
      Paloma Pay
    </span>
  </div>
</WithStyles(ForwardRef(Container))>
`;
